guide/book.toml

```diff
@@ -2,3 +2,6 @@
 title = "PyO3 user guide"
 description = "PyO3 user guide"
 author = "PyO3 Project and Contributors"
+
+[preprocessor.pyo3_version]
+command = "python guide/pyo3_version.py"
```

Comment:
Ah, but maybe this should be `python3 guide/pyo3_version.py` for old Linux users...
I'm sorry but I came up with this idea when pressing the merge button.